### ISSUE TITLE
Incompatibility with Tomcat 8

### ISSUE THREAD

**Issue by jrwiebe** [iipc/openwayback#169]
In my testing, openwayback runs only on Tomcat 6 or 7, and not Tomcat 8. When I install the webapp 
and restart Tomcat 8, I get the following in my `catalina.out`:

```
Sep 17, 2014 4:55:38 PM org.apache.catalina.core.StandardContext filterStart
SEVERE: Exception starting filter RequestFilter
java.lang.NullPointerException
        at java.io.File.<init>(File.java:251)
        at org.archive.wayback.util.webapp.RequestFilter.init(RequestFilter.java:72)
        at org.apache.catalina.core.ApplicationFilterConfig.initFilter(ApplicationFilterConfig.java:279)
        at org.apache.catalina.core.ApplicationFilterConfig.getFilter(ApplicationFilterConfig.java:260)
        at org.apache.catalina.core.ApplicationFilterConfig.<init>(ApplicationFilterConfig.java:105)
        at org.apache.catalina.core.StandardContext.filterStart(StandardContext.java:4594)
        at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5201)
        at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:150)
        at org.apache.catalina.core.ContainerBase.addChildInternal(ContainerBase.java:724)
        at org.apache.catalina.core.ContainerBase.addChild(ContainerBase.java:700)
        at org.apache.catalina.core.StandardHost.addChild(StandardHost.java:714)
        at org.apache.catalina.startup.HostConfig.deployWAR(HostConfig.java:919)
        at org.apache.catalina.startup.HostConfig$DeployWar.run(HostConfig.java:1704)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
        at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
        at java.util.concurrent.FutureTask.run(FutureTask.java:166)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
        at java.lang.Thread.run(Thread.java:722)

17-Sep-2014 16:55:38.823 SEVERE [localhost-startStop-1] org.apache.catalina.core.StandardContext.startInternal Error filterStart
```

I am running Tomcat 8.0.12 on OS X 10.9.4 with Java 1.7. I have no problems with Tomcat 6 or 7. I took a look at the Tomcat and the openwayback code and I can't figure out why RequestFilter.init is being passed a null value.

**Comment by dmolesUC3** [comment]
The workaround for this is to change `logging-config-path` and `config-path` to absolute paths in `web.xml`: see https://github.com/iipc/openwayback/issues/196

**Comment by PsypherPunk** [comment]
Fixed in #198.
